### PR TITLE
More practice tasks

### PR DESCRIPTION
**Pull request by GeorgiErmenkov**

### CODE REVIEW

File: one-file-tasks/qeustion64.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#Then, the output of the program should be:
+#3.55
+#In case of input data being supplied to the question, it should be assumed to be a console input.
+#Hints:
+#Use float() to convert an integer to a float
+
+num = int(input())
+sum = 0.0
+for n in range(1,num+1):
+    sum += float(n)/float(n+1)
```
Comment (dnstufff) [inline one-file-tasks/qeustion64.py]: Do you need casting here?

File: one-file-tasks/question58.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+#In case of input data being supplied to the question, it should be assumed to be a console input.
+#Hints:
+#Use \w to match letters.
+
+import re
+
+email = input()
+matchit = re.match('(\w+)@(\w+).(\w+)',email)
+print(matchit.group(1))
+
```
Comment (dnstufff) [inline one-file-tasks/question58.py]: Nice!

File: one-file-tasks/question60.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+#Then, the output of the program should be:
+#['2', '3']
+#In case of input data being supplied to the question, it should be assumed to be a console input.
+#Hints:
+#Use re.findall() to find all substring using regex.
+
+import re
+
+words = input()
+print(re.findall('\d+',words))
```
Comment (dnstufff) [inline one-file-tasks/question60.py]: Nice! Add space after ","

File: one-file-tasks/question68.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+i=0
+while (i <= m):
+    if i % 2 == 0 and i != m:
+        answer += str(i) + ','
+    elif i % 2 == 0 and i == m:
+        answer += str(i)
+    else:
+        pass
+    i += 1
+print(answer)
```
Comment (dnstufff) [inline one-file-tasks/question68.py]: Add blank line in the end

File: one-file-tasks/question68.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+#Use yield to produce the next value in generator.
+#In case of input data being supplied to the question, it should be assumed to be a console input.
+
+
+
+#I  made it with loops
+
+n = int(input())
+val_list=[]
+for i in range(0,n+1):
```
Comment (dnstufff) [inline one-file-tasks/question68.py]: Add space after ","

File: one-file-tasks/question68.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+#Hints:
+#Use yield to produce the next value in generator.
+#In case of input data being supplied to the question, it should be assumed to be a console input.
+
+
+
+#I  made it with loops
+
+n = int(input())
+val_list=[]
```
Comment (dnstufff) [inline one-file-tasks/question68.py]: Add space around "=". Install python linter for VScode

File: one-file-tasks/question68.py [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+for i in range(0,n+1):
+    if i % 2==0:
+        val_list.append(str(i))
+    else:
```
Comment (dnstufff) [inline one-file-tasks/question68.py]: No need for this "else"

File: one-file-tasks/question68.py [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        pass
+
+print(','.join(val_list))
+
+#Aprroach with while loop
+
+m = int(input())
+answer=''
```
Comment (dnstufff) [inline one-file-tasks/question68.py]: Add space around "="

File: one-file-tasks/question68.py [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+i=0
+while (i <= m):
+    if i % 2 == 0 and i != m:
+        answer += str(i) + ','
+    elif i % 2 == 0 and i == m:
+        answer += str(i)
+    else:
```
Comment (dnstufff) [inline one-file-tasks/question68.py]: No need for this "else"

File: one-file-tasks/question68.py [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+        pass
+
+print(','.join(val_list))
+
+#Aprroach with while loop
+
+m = int(input())
+answer=''
+i=0
+while (i <= m):
```
Comment (dnstufff) [inline one-file-tasks/question68.py:30]: Better use ', '.join() as in the "for" example

**Review by dnstufff — CHANGES_REQUESTED** [review]

File: one-file-tasks/question68.py (hunk 10 above)
Comment (GeorgiErmenkov) [reply one-file-tasks/question68.py:30]: Just wanted to do it differently

File: one-file-tasks/Classes-ex1.py [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+    def __init__(self, name, max_speed, mileage):
+        self.name = name
+        self.max_speed = max_speed
+        self.mileage = mileage
+
+    def seating_capacity(self, capacity):
+        return f"The seating capacity of a {self.name} is {capacity} passengers. And is {self.size}"
+
+class Bus(Vehicle):
+    def __init__(self,name,max_speed, milage, size='big'):
```
Comment (dnstufff) [inline one-file-tasks/Classes-ex1.py]: Spaces after ",". Install a linter on VScode

File: one-file-tasks/Classes-ex1.py [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+        super().__init__(name, max_speed, milage)
+        self.size=size
+    
+    def seating_capacity(self, capacity):
```
Comment (dnstufff) [inline one-file-tasks/Classes-ex1.py]: What's the point of this method? You have it already defined in the parent class and you can use it without defining it again in the child class.

File: one-file-tasks/Classes-ex1.py [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+        return super().seating_capacity(capacity)
+
+School_bus = Bus("School Volvo", 150, 10000)
+print(School_bus.seating_capacity(50))
```
Comment (dnstufff) [inline one-file-tasks/Classes-ex1.py]: New Line at the end. Install a linter for VScode.

File: one-file-tasks/Classes-ex2.py [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+
+    def fare(self):
+        return self.capacity * 100
+
+class Bus(Vehicle):
+    def fare(self):
+        return self.capacity * 110
+
+School_bus = Bus("School Volvo", 12, 100)
+print("Total Bus fare is:", School_bus.fare())
```
Comment (dnstufff) [inline one-file-tasks/Classes-ex2.py:24]: New line

File: one-file-tasks/Functions-ex1.py [hunk 15]
```diff
@@ -0,0 +1,39 @@
+#Exercise Question 4: Create a function showEmployee() in such a way that it should accept employee 
+# name, and it’s salary and display both, and if the salary is missing in function call it should 
+# show it as 9000
+
+def showEmployee(name,salary=9000):
```
Comment (dnstufff) [inline one-file-tasks/Functions-ex1.py]: Spaces

File: one-file-tasks/Functions-ex1.py [hunk 16]
```diff
@@ -0,0 +1,39 @@
...
+    print('Employee', name, 'salary is',salary)
```
Comment (dnstufff) [inline one-file-tasks/Functions-ex1.py]: Spaces

File: one-file-tasks/Functions-ex1.py [hunk 17]
```diff
@@ -0,0 +1,39 @@
...
+
+showEmployee('John')
+
+#Exercise Question 5: Create an inner function to calculate the addition in the following way
+#Create an outer function that will accept two parameters a and b
+#Create an inner function inside an outer function that will calculate the addition of a and b
+#At last, an outer function will add 5 into addition and return it
+
+def add5(a,b):
```
Comment (dnstufff) [inline one-file-tasks/Functions-ex1.py]: Spaces after "," "+"... and so on. Install a linter for VScode

File: one-file-tasks/Functions-ex1.py [hunk 18]
```diff
@@ -0,0 +1,39 @@
...
+        return a+b
+    res=add(a,b) + 5
+    return res
+
+result=add5(5,10)
+print(result)
+
+#Exercise Question 6: Write a recursive function to calculate the sum of numbers from 0 to 10
+
+def sumof(num):
```
Comment (dnstufff) [inline one-file-tasks/Functions-ex1.py:26]: Impressive!

File: one-file-tasks/Functions-ex1.py [hunk 19]
```diff
@@ -0,0 +1,39 @@
...
+    if num:
```
Comment (dnstufff) [inline one-file-tasks/Functions-ex1.py]: if num > 0

File: one-file-tasks/Functions-ex2.py [hunk 20]
```diff
@@ -0,0 +1,21 @@
...
+        if n % 2 == 0:
+            evens_list.append(n)
+    return evens_list
+
+print(evens(4,30))
+
+#Exercise Question 9: Return the largest item from the given list
+import math
+def largest_item(l):
+    large_num = max(l)
```
Comment (dnstufff) [inline one-file-tasks/Functions-ex2.py:16]: Тарикат :D

File: one-file-tasks/question 69.py [hunk 21]
```diff
@@ -0,0 +1,22 @@
...
+    for i in range(n+1):
+        if i % 5 == 0 and i % 7 == 0:
+            yield i
+
+val_list = []
+n=int(input())
+for j in gener(n):
+    val_list.append(str(j))
+print(','.join(val_list))
+
```
Comment (dnstufff) [inline one-file-tasks/question 69.py]: Impressive!

**Review by dnstufff — CHANGES_REQUESTED** [review]
Amazing progress!

**Review by dnstufff — APPROVED** [review]

**Pull request merged**